src/components/NavigationBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { BottomNavigation, BottomNavigationItem } from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+
+import FontIcon from 'material-ui/FontIcon';
+const settingsIcon = <FontIcon className="material-icons">settings</FontIcon>;
+const boardIcon = <FontIcon className="material-icons">view_module</FontIcon>;
+const keyboardIcon = <FontIcon className="material-icons">keyboard</FontIcon>;
+
+class NavigationBar extends PureComponent {
+  render() {
+    return (
+      <Paper zDepth={1}>
+        <BottomNavigation selectedIndex={this.props.selectedIndex}>
+          <BottomNavigationItem
+            label={this.props.intl.formatMessage({ id: 'cboard.containers.App.bottomNavigationItem.settings' })}
```

Comment:
can we cache this.props to a variable? as `this` is not minified while creating bundle